mavros/src/plugins/global_position.cpp

```diff
@@ -156,6 +159,9 @@ class GlobalPositionPlugin : public plugin::PluginBase {
 	void handle_gps_raw_int(const mavlink::mavlink_message_t *msg, mavlink::common::msg::GPS_RAW_INT &raw_gps)
 	{
 		auto fix = boost::make_shared<sensor_msgs::NavSatFix>();
+		auto sat_cnt = boost::make_shared<std_msgs::UInt32>();
+		sat_cnt->data = raw_gps.satellites_visible;
+		raw_sat_pub.publish(sat_cnt);
```

Comment:
I'd prefer to place that new code to the tail of the function.

Reply:
Sounds reasonable! See latest commit.